app/components/events/view/export/api-response.js

```diff
@@ -29,3 +65,23 @@ function syntaxHighlight(json) {
     return `<span class=" ${cls}"> ${match}</span>`;
   });
 }
+
+function buildURL(baseURL, toggleSwitches) {
```

Comment:
Use the below method to generate the URL

https://github.com/fossasia/open-event-frontend/blob/7ea046d0470108c87529c1a3a7f50d317aea4247/app/utils/url.js#L35-L47

Reply:
@niranjan94 I am having trouble using this method since I need to check if I need to add `include` or not separately and hence I iterate over the `toggleSwitches` object for this. During this iteration itself, I can form the url.

Reply:
Wouldn't something like

Reply:
Wouldn't something like

Reply:
Wouldn't something like 

```js
function buildURL(baseURL, toggleSwitches) {
  let newUrl = baseURL;
  const include = [];

  for (const key in toggleSwitches) {
    if (toggleSwitches.hasOwnProperty(key)) {
      toggleSwitches[key] && include.push(key);
    }
  }

  return buildUrlViaQueryString(newUrl, {
    include: include.length > 0 ? include : undefined
  }, true)
}
```

work ?

Reply:
Also move this entire function into the component definition and directly use `this.get('toggleSwitches')`